website/source/docs/job-specification/service.html.md

```diff
@@ -104,6 +104,9 @@ does not automatically enable service discovery.
   `address_mode="driver"`. Numeric ports may be used when in driver addressing
    mode.
 
+   Docker and IPv6 containers: This setting is required if you want to register 
+   the port of the (IPv6) service. See [below for examples.](#IPv6 docker containers)
```

Comment:
If we set `auto=true` when `use_ipv6_address=true` this will no longer be necessary.